src/Tribe/Main.php

```diff
@@ -904,7 +904,7 @@ public function action_expire_archive_slug_conflict_notice( $new_status, $old_st
 			}
 
 			$post = get_post( $post );
-			$archive_slug = Tribe__Settings_Manager::get_option( 'eventsSlug', 'events' );
+			$archive_slug = _x( Tribe__Settings_Manager::get_option( 'eventsSlug', 'events' ), 'Archive Events Slug', 'the-events-calendar' );
```

Comment:
Here and other places where we're passing the result of a function call to `_x()` I feel like a translators note could be useful to help us and the community both (to reduce questions/complaints and to let the team quickly discover it's deliberate when questions come in regardless). 

Maybe something like:

`/* translators: for compatibility with WPML and other multilingual plugins, not to be translated directly */`

Reply:
Thanks, excellent idea ;)